.github/workflows/cicd-1-pull-request.yaml

```diff
@@ -39,7 +46,7 @@ jobs:
           echo "nodejs_version=$(grep "^nodejs" .tool-versions | cut -f2 -d' ')" >> $GITHUB_OUTPUT
           echo "python_version=$(grep "^nodejs" .tool-versions | cut -f2 -d' ')" >> $GITHUB_OUTPUT
           echo "terraform_version=$(grep "^terraform" .tool-versions | cut -f2 -d' ')" >> $GITHUB_OUTPUT
-          echo "version=$(head -n 1 .version 2> /dev/null || echo unknown)" >> $GITHUB_OUTPUT
+          echo "version=${{steps.semantic.outputs.new_release_version}}" >> $GITHUB_OUTPUT
```

Comment:
I'm not sure we need the version at this stage of the workflow as commits could be being made ahead of us?